engine/apps/grafana_plugin/views/sync.py

```diff
@@ -49,19 +51,20 @@ def post(self, request: Request) -> Response:
             },
         )
 
-    def get(self, request: Request) -> Response:
+    def get(self, _request: Request) -> Response:
         stack_id = self.instance_context["stack_id"]
         org_id = self.instance_context["org_id"]
-
         token_ok = False
+
         try:
             organization = Organization.objects.get(stack_id=stack_id, org_id=org_id)
-            if organization.api_token_status == Organization.API_TOKEN_STATUS_PENDING:
```

Comment:
What happened to this check? In the post handler above `plugin_sync_organization_async.apply_async` is started and will be in PENDING state until complete. How do we know users and teams are populated?